### PR TITLE
flush db after the tests

### LINKED ISSUE #31: dataStore test runs properly only once
It seems that dataStore tests do not delete from database...

### PR DESCRIPTION
**Pull request by zalun**
fixes #31

### CODE REVIEW

File: test/dataStore.js [hunk 1]
```diff
@@ -238,5 +238,9 @@ describe('DB with redis backend', () => {
     });
   });
 
+  after(() => {
```
Comment (delapuente) [inline test/dataStore.js]: You can simplify to:

``` js
after(() => dataStore.flush());
```

File: dataStore/backend-redis.js [hunk 2]
```diff
@@ -109,10 +109,19 @@ function configure(opts) {
   });
 }
 
+function flush() {
+  return new Promise(resolve => {
+    client.flushdb(() => {
```
Comment (delapuente) [inline dataStore/backend-redis.js:114]: Don't you need to check for the error? At least, to not silence it. If you do, you could _promisify_ this method.

Comment (zalun) [inline dataStore/backend-redis.js:114]: Documentation of redis says "it never fails"

Comment (delapuente) [inline dataStore/backend-redis.js:114]: Then you could do:

``` js
client.flushdb(resolve); // redis says it never fails
```

**Pull request merged**